lib/ts/recipe/dashboard/api/analytics.ts

```diff
@@ -71,9 +71,12 @@ export default async function analyticsPost(
         };
     }
 
-    const { apiDomain, websiteDomain, appName } = options.appInfo;
+    const { apiDomain, getOrigin: websiteDomain, appName } = options.appInfo;
     const data = {
-        websiteDomain: websiteDomain.getAsStringDangerous(),
+        websiteDomain: websiteDomain({
+            request: undefined,
+            userContext: {},
```

Comment:
create default user context here. Not an empty object. There should be no `{}` for user context in any of your changes.